target/linux/realtek/files-6.12/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -3917,6 +3925,21 @@ static struct phy_driver rtl83xx_phy_driver[] = {
 		.resume		= genphy_resume,
 		.read_status	= rtl9300_read_status,
 	},
+	{
+		PHY_ID_MATCH_MODEL(PHY_ID_RTL8224),
+		.name		= "REALTEK RTL8224",
+		.features	= PHY_10GBIT_FEATURES,
+		.flags		= PHY_POLL_CABLE_TEST,
```

Comment:
There is no cable test functionality. And I would have expected the usage of `rtl821x_read_page` and `rtl821x_write_page`